ffxivdpscalc/classimpl/sch.py

```diff
@@ -0,0 +1,47 @@
+import math
+
+from ffxivdpscalc.calc import CharacterStats
+
+
+class SchCharacterStats(CharacterStats):
+    r2_potency = 200
+    b3_potency = 290
+    bio_potency = 70
+    ed_potency = 100
+
+    def __init__(self, job, wd, mainstat, det, crit, dh, speed, ten, pie):
+        super().__init__(job, wd, mainstat, det, crit, dh, speed, ten, pie)
+        
+    # hard coded for a ~180 second cycle, actual length calculated by get_cycle
+    # todo: extend this for variable length
+    def pps_spreadsheet_port(self, caster_tax, num_ed_per_min):
```

Comment:
as discussed via discord i need to think about this in more detail because of sch rotation stuff. pls make the other changes while my brain wakes up

Reply:
Going to punt on this to the next PR, where we try to unfurl the math and make it more extensible.